src/compton/Compton.hh

```diff
@@ -60,9 +60,12 @@ private:
   //! Shared pointer to an electron interpolation object:
   std::shared_ptr<etemp_interp> ei;
 
+  //! Shared pointer to an llnl interpolation object:
+  std::shared_ptr<llnl_interp> llnli;
+
 public:
   //! Constructor for an existing multigroup library
-  explicit Compton(const std::string &filehandle);
+  Compton(const std::string &filehandle, const bool llnl_style = false);
```

Comment:
Leave the `explicit` keyword here since you still don't really want an automatic promotion path from `std::string` to `Compton`.

Reply:
Ahh, okay. So the two-argument constructor can still be promoted if the second variable isn't passed (and the default is used instead)? I didn't know that!